# HG changeset patch
# User pm
# Date 1551804274 -3600
#      Tue Mar 05 17:44:34 2019 +0100
# Node ID 82a5c54050eb32ac4eb0b651869998acd541c78b
# Parent  9d795c8d30cc163d3fd3e332c9d5cde8c1b250a0
Ticket mantis 0005262: Modifier le titre de la page d'accueil

diff --git a/comm-esr60/comm/mailnews/base/content/msgAccountCentral.js b/comm-esr60/comm/mailnews/base/content/msgAccountCentral.js
--- a/comm-esr60/comm/mailnews/base/content/msgAccountCentral.js
+++ b/comm-esr60/comm/mailnews/base/content/msgAccountCentral.js
@@ -40,7 +40,7 @@
             let acctName = msgFolder.prettyName;
             // Display and collapse items presented to the user based on account type
             title = messengerBundle.getFormattedString("acctCentralTitleFormat",
-                                                       [brandName, acctType, acctName]);
+                                                       [acctType, acctName]);
         } else {
           // If there is no selectedServer, we are in a brand new profile with
           // no accounts - show the create account rows.
diff --git a/fr-esr60/fr/mail/chrome/messenger/messenger.properties b/fr-esr60/fr/mail/chrome/messenger/messenger.properties
--- a/fr-esr60/fr/mail/chrome/messenger/messenger.properties
+++ b/fr-esr60/fr/mail/chrome/messenger/messenger.properties
@@ -309,7 +309,7 @@
 # accountCentral
 mailnews.account_central_page.url=chrome://messenger/content/msgAccountCentral.xul
 # LOCALIZATION NOTE(acctCentralTitleFormat): %1$S is brand, %2$S is account type, %3$S is account name
-acctCentralTitleFormat=%1$S - %2$S de %3$S
+acctCentralTitleFormat=%1$S de %2$S
 mailAcctType=Courrier
 newsAcctType=Groupes
 feedsAcctType=Nouvelles et Blogs
